Sources/SwifterSwift/SwiftStdlib/ArrayExtensions.swift

```diff
@@ -146,4 +146,13 @@ public extension Array where Element: Equatable {
         }
     }
 
+    /// SwifterSwift: Returns a set with all duplicate elements removed using KeyPath to compare.
+    ///
+    /// - Parameter path: Key path to compare, the value must be Hashable.
+    /// - Returns: a set of unique elements.
```

Comment:
Now you're returning an array
```suggestion
    /// - Returns: an array of unique elements.
```